src/System.Composition.Convention/src/System.Composition.Convention.csproj

```diff
@@ -61,5 +61,21 @@
       <Name>System.Composition.AttributedModel</Name>
     </ProjectReference>
```

Comment:
Should we also go through all of these src projects and remove these P2P references?